### PR TITLE
let zowe launch script to handle configure

### PR DESCRIPTION
**Pull request by jackjia-ibm**
Signed-off-by: Jack (T.) Jia <jack-tiefeng.jia@ibm.com>

Part of fixes for https://github.com/zowe/zowe-install-packaging/issues/1769

### CODE REVIEW

**Comment by jackjia-ibm** [comment]
The integration test has been done https://github.com/zowe/zowe-install-packaging/pull/1849#issuecomment-746265697. This PR depends on https://github.com/zowe/zowe-install-packaging/pull/1849, must be merged after that. Please let me know if there are any questions. Thanks.

**Review by stevenhorsman — APPROVED** [review]

File: data-sets-zowe-server-package/src/main/resources/bin/configure.sh [hunk 1]
```diff
@@ -1,77 +0,0 @@
...
-# https://www.eclipse.org/legal/epl-v20.html
-#
-# SPDX-License-Identifier: EPL-2.0
-#
-# Copyright IBM Corporation 2019, 2020
-################################################################################
-
-# Source main utils script
-. ${ROOT_DIR}/bin/utils/utils.sh
-ensure_java_is_on_path
```
Comment (stevenhorsman) [inline data-sets-zowe-server-package/src/main/resources/bin/configure.sh:15]: Sorry, something I noticed after reviewing the APIML components - this check was kept in configure there. Is there a reason for the different behaviour in data-sets and jobs?

Comment (jackjia-ibm) [reply data-sets-zowe-server-package/src/main/resources/bin/configure.sh:15]: Yes this is in prepare-environments https://github.com/zowe/zowe-install-packaging/blob/staging/bin/internal/prepare-environment.sh#L142. I'm kind mixed due to the fact of we may want to make java/node optional.... I think I should also delete configure.sh on api-layer.

Comment (jackjia-ibm) [reply data-sets-zowe-server-package/src/main/resources/bin/configure.sh:15]: Updated api-layer https://github.com/zowe/api-layer/pull/1002/commits/6f073ac532349640a2daf706e62c922702d17fc3

**Pull request merged**